apps/journeys-admin/src/components/TemplateView/TemplateViewHeader/TemplateViewHeader.spec.tsx

```diff
@@ -155,8 +155,8 @@ describe('TemplateViewHeader', () => {
     expect(getAllByRole('link', { name: 'Preview' })[0]).toBeInTheDocument()
   })
 
-  it('should show featured date if journey is featured', () => {
-    const { getAllByTestId } = render(
+  it('should show created at date', () => {
+    const { getAllByText } = render(
```

Comment:
Use getBy for single elements, or check for length if there are more than 1
```suggestion
    const { getByText } = render(
```

Reply:
this is one of those elements that have xs: hidden sm: block so they still show up twice in the dom. Maybe a better way I can assert this is     `expect(getAllByText('November 2021')).toHaveLength(2)`